apps/core/src/components/stake/StakedCard.tsx

```diff
@@ -61,6 +76,21 @@ export function StakedCard({
             <CardBody
                 title={validatorMeta?.name || '--'}
                 subtitle={`${principalStaked} ${symbol}`}
+                icon={
+                    activeButNotInTheCommittee ? (
+                        <RewardsOff className="text-warning-60" />
+                    ) : inactiveValidator ? (
+                        <Warning className="text-error-30" />
+                    ) : null
+                }
+                tooltipText={
+                    activeButNotInTheCommittee
+                        ? 'This validator is not part of the next epoch, so you will not earn rewards.'
+                        : inactiveValidator
+                          ? 'This validator is inactive, so you will not earn rewards.'
+                          : ''
+                }
```

Comment:
```suggestion
                tooltipText={
                    activeButNotInTheCommittee
                        ? 'This validator is active but not in the current committee, so it’s not earning rewards right now. It may earn in future epochs.'
                        : inactiveValidator
                          ? 'This validator is inactive and not earning rewards. It must become active and join the committee to earn in the future.'
                          : ''
                }
```